chaindataloader/binary_test.go

```diff
@@ -0,0 +1,205 @@
+package chaindataloader
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/btcsuite/btcd/wire"
+	"github.com/lightninglabs/neutrino/headerfs"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	testDataDir = "testdata/"
+)
+
+type testSideLoadConfig struct {
+	name         string
+	filePath     string
+	endHeight    uint32
+	startHeight  uint32
+	network      wire.BitcoinNet
+	tipHeight    uint32
+	err          bool
```

Comment:
These should be `error`. Then we can use `errors.Is` to make sure the proper error is returned.